tests/unit/malware/test_setup_patterns.py

```diff
@@ -0,0 +1,243 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Can you create a new directory under `malware` called `checks` and put this file in there?